locale/tr.po

```diff
@@ -1558,99 +1552,87 @@ msgstr "&Komut"
 
 #: src/BatchCommandDialog.cpp
 msgid "&Edit Parameters"
-msgstr "&Parametreleri Düzenle"
+msgstr "Parametreleri &Düzenle"
 
 #: src/BatchCommandDialog.cpp
 msgid "&Use Preset"
-msgstr "Hazır Ayar K&ullanılsın"
+msgstr "Ön Ayarı &Kullan"
 
 #: src/BatchCommandDialog.cpp
 msgid "&Parameters"
 msgstr "&Parametreler"
 
 #: src/BatchCommandDialog.cpp
 msgid "&Details"
-msgstr ""
+msgstr "&Ayrıntılar"
 
 #: src/BatchCommandDialog.cpp
-#, fuzzy
 msgid "Choose command"
-msgstr "Kom&ut seçin"
+msgstr "Komut seçin"
 
 #: src/BatchCommands.cpp
-#, fuzzy
 msgid "MP3 Conversion"
-msgstr "Gerçek Zamanlı Dönüştürme"
+msgstr "MP3 Dönüştürme"
 
 #: src/BatchCommands.cpp
-#, fuzzy
 msgid "Fade Ends"
-msgstr "Artarak Gir"
+msgstr "Bitişte Ses seviyesini azalt"
 
 #: src/BatchCommands.cpp
-#, fuzzy
 msgid "Import Macro"
-msgstr "Ham Veri İçe Aktarma"
+msgstr "Makroyu İçe Aktar"
 
 #: src/BatchCommands.cpp
-#, fuzzy, c-format
+#, c-format
 msgid "Macro %s already exists. Would you like to replace it?"
-msgstr ""
-"\"%s\" bulunamadı.\n"
-"\n"
-"Oluşturmak istiyor musunuz?"
+msgstr "%s makrosu zaten var. Değiştirmek ister misiniz?"
 
 #: src/BatchCommands.cpp
-#, fuzzy
 msgid "Export Macro"
-msgstr "MIDI Dışa Aktar"
+msgstr "Makroyu Dışa Aktar"
 
 #: src/BatchCommands.cpp
-#, fuzzy
 msgid "Effect"
-msgstr "Etkiler"
+msgstr "Efekt"
 
 #: src/BatchCommands.cpp
-#, fuzzy
 msgid "Menu Command (With Parameters)"
-msgstr "&Parametreleri Düzenle"
+msgstr "Menü Komutu (Parametreli)"
 
 #: src/BatchCommands.cpp src/FileNames.cpp
 #, c-format
 msgid "(%s)"
-msgstr ""
+msgstr "(%s)"
 
 #: src/BatchCommands.cpp
 msgid "Menu Command (No Parameters)"
-msgstr ""
+msgstr "Menü Komutu (Parametre Yok)"
 
 #. i18n-hint: %s will be replaced by the name of an action, such as "Remove Tracks".
 #: src/BatchCommands.cpp src/CommonCommandFlags.cpp
 #, c-format
 msgid "\"%s\" requires one or more tracks to be selected."
-msgstr ""
+msgstr "\"%s\" bir veya daha fazla parçanın seçilmesini gerektiriyor."
 
 #: src/BatchCommands.cpp
 #, c-format
 msgid "Your batch command of %s was not recognized."
-msgstr "%s yığın komutunuz anlaşılamadı."
+msgstr "%s toplu komutunuz tanınmadı."
```

Comment:
anlaşılamadı bu bağlamda doğru. batch için toplu iş olarak kullanılır genelde. tanınmadı doğrudan çeviri oluyor.